deploy/03_resolve_uniswap_v2_router.ts

```diff
@@ -1,6 +1,8 @@
 import { HardhatRuntimeEnvironment, HardhatNetworkConfig } from "hardhat/types"
 import { DeployFunction } from "hardhat-deploy/types"
 
+// TODO: THIS DEPLOYMENT CAN BE DONE ONLY IF THERE IS A UNISWAP POOL WITH
```

Comment:
Let's drop all deployment scripts related to v1 risk manager. T coverage pool will not support this way of claiming the coverage, the only option will be the multisig claim implemented in #216. Details in this [proposal](https://forum.threshold.network/t/tip-043-coverage-pool-migration/465). 

Let's also add `@dev` comment to v1 risk-manager-related contracts that the given contract is used only for KEEP coverage pool in v1 that got sunset, and drop a link to TIP-043.

Reply:
Partially done in [82173bc](https://github.com/keep-network/coverage-pools/pull/203/commits/82173bc726c4622bda153c68c16c9aade559484e)

Reply:
Added a dev note only for one contract. If the wording looks good to you, I'll add it in other contracts. [7b22857](https://github.com/keep-network/coverage-pools/pull/203/commits/7b228572426d60aa5c4dd61ca1a19987baed676b)